verify-merge.py

```diff
@@ -5,27 +5,99 @@
 import glob
 import sys
 
-GIT = os.getenv('GIT','git')
-GPG = os.getenv('GPG','gpg')
+GIT = os.getenv('GIT', 'git')
+GPG = os.getenv('GPG', 'gpg')
+GITIAN_PUBKEYS_DIR = os.getenv('GITIAN_PUBKEYS_DIR', 'gitian-pubkeys')
 
 def verify():
     global args, workdir
     if args.import_keys:
-        os.chdir('gitian-pubkeys')
-        print('Importing gpg pubkeys...')
-        keys = [f for f in glob.glob('*.asc', recursive=False)]
-        for key in keys:
-            subprocess.check_call([GPG, '--import', key])
-        os.chdir('../')
+        import_gpg_keys()
     if args.refresh_keys:
-        print('Refreshing pubkeys...')
-        subprocess.check_call([GPG, '--refresh'])
+        refresh_gpg_keys()
+    assert_files = get_assert_file_list()
+    verify_gpg_sigs(assert_files)
+    verify_checksums(assert_files)
+    print('All checks passed.')
+    os.chdir(workdir)
+
+def main():
+    global args, workdir
+    args = get_parsed_args()
+    workdir = os.getcwd()
+    if args.pull_id != None:
+        pull_request()
+    else:
+        verify()
+
+def get_parsed_args():
+    global args
```

Comment:
This variable declaration is not needed.

Reply:
Thanks, removed.

Some of the other `global` declarations are also not necessary because the global variable _isn't being modified_, but still nice to have it there for readability I suppose.